scripts/create-component/create-component.ts

```diff
@@ -169,13 +167,15 @@ module.exports = (plop: NodePlopAPI) => {
             shell: true,
           });
           if (yarnResult.status !== 0) {
-            console.error('Something went wrong with building. Please check previous logs for details');
+            console.error(chalk.red.bold('Something went wrong with building. Please check previous logs for details'));
             process.exit(1);
           }
           return 'Component compiled!';
         },
-        '\nCreated new component! Please check over it and ensure wording and included files ' +
-          'make sense for your scenario.',
+        stripIndent(`
```

Comment:
I think the previous version of this log was actually better, because at least on a halfway-decent console it should automatically wrap based on the actual console width. With this change you could end up with awkward line breaks if the width isn't as expected.